clouds/aws/aws_utils.py

```diff
@@ -139,3 +139,34 @@ def delete_bucket(bucket_name: str, boto_client) -> None:
         f"Bucket {bucket_name} not deleted",
         json.dumps(response, defualt=str, indent=4),
     )
+
+
+def aws_region_names():
+    """
+    Lists AWS regions
+
+    Returns:
+        list: list of AWS regions name
+
+    """
+    regions = ec2_client().describe_regions()["Regions"]
+    return [region["RegionName"] for region in regions]
+
+
+def get_least_crowded_aws_vpc_region(region_list):
+    """
+    Selects region with the least number of VPCs.
+
+    Args:
+        region_list: list of region names
+
+    Returns:
+        str: region name with have the least number of VPCs
+
+    """
+    region, vpcs = 0, 0
+    for _region in region_list:
+        if (num_vpcs := len(ec2_client(region_name=_region).describe_vpcs()["Vpcs"])) <= vpcs:
```

Comment:
@myakove I added `<=` because initially `vpcs is 0` and num_vpcs can never be less than 0, so this if condition will never become true for `num_vpcs < vpcs` and that's why I'm checking if it can be either `<= 0 or vpcs`

Reply:
I see, -1 will solve it :)
But this is OK